RELEASE/scripts/autoscend/auto_familiar.ash

```diff
@@ -855,7 +861,11 @@ void acquireFamiliars()
 	{
 		return;
 	}
-	
+	if(in_bhy()) // BHY can't use either barrel mimic or blood-faced volleyball
+	{
+		return;
+	}
+
```

Comment:
Actually I tested this in BHY and it works. Hatching familiars works just fine in BHY.
You won't be able to use those familiars in BHY, but that is no reason to avoid getting them if you are somehow finding yourself in in postronin BHY.
so this extra code is unnecessary

Reply:
I don't think BHY lets you make the blood-faced volleyball, and ended up just accumulating hundreds of turns of Bloody Hand as a result.  I will review and edit the code accordingly if that's the case

Reply:
> I don't think BHY lets you make the blood-faced volleyball, and ended up just accumulating hundreds of turns of Bloody Hand as a result. I will review and edit the code accordingly if that's the case

hmmm. actually you know what. I was wrong. It is a use() command there.
although instead of of this in_bhy() check. I think below it should replace
```
if(!have_familiar($familiar[Blood-Faced Volleyball]) && item_amount($item[blood-faced volleyball]) == 0 && my_meat() > meatReserve() + 1500)
```
with
```
if(!have_familiar($familiar[Blood-Faced Volleyball]) && item_amount($item[blood-faced volleyball]) == 0 && my_meat() > meatReserve() + 1500 && auto_is_valid($item[seal tooth]) && auto_is_valid($item[volleyball]))
```